es6/tests/base.js

```diff
@@ -924,3 +925,38 @@ describe("Serialization", function() {
 		JSON.stringify(doc);
 	});
 });
+
+describe("Docxtemplater v4 tests", function() {
+	it("should work with new constructor", function() {
+		let isModuleCalled = false;
+
+		const module = {
+			optionsTransformer(options) {
+				isModuleCalled = true;
+				return options;
+			},
+		};
+
+		createDocV4("tag-example.docx", { modules: [module] });
+		expect(isModuleCalled).to.equal(true);
+	});
+
+	it("should throw error when modules passed is not an array", function() {
+		expect(createDocV4.bind(this, "tag-example.docx", { modules: {} })).to.throw(
+			"Please pass modules as an array"
+		);
+	});
+
+	it("should render correctly using the new constructor", () => {
+		const docForZip = createDoc("tag-example.docx");
+		const zip = new PizZip(docForZip.loadedContent);
+		const doc = new Docxtemplater(zip);
```

Comment:
Isn't this the same as : 

```
const doc = createDocV4("tag-example.docx") 
```

Reply:
Yes, My thoughts here are: I get zip from a known document and then validate the creating a document with the new constructor. This lets me validate that the new constructor is loading the zip correctly.

Reply:
Ok, I agree with part of it, the problem is that it does'nt make much sense to create another docxtemplater instance just to retrieve the zip file.

Can you add instead a new method in the test utils : getZip("tag-example.docx"), and write : 

```
const doc = new Docxtemplater(getZip("tag-example.docx"))
```

Reply:
Sure, I will do it.